scripts/insolard/generate_initial_data.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -e
+
+# configurable vars
+INSOLAR_ARTIFACTS_DIR=${INSOLAR_ARTIFACTS_DIR:-".artifacts"}/
+LAUNCHNET_BASE_DIR=${LAUNCHNET_BASE_DIR:-"${INSOLAR_ARTIFACTS_DIR}launchnet"}/
```

Comment:
set slash between  INSOLAR_ARTIFACTS_DIR and launchnet as `${INSOLAR_ARTIFACTS_DIR}/launchnet"}/`

POSIX systems able to work correctly even if you pass double slash in the path. for example `ls -al /tmp//folder/`

Reply:
done